airbyte-integrations/connectors/source-zendesk-sunshine/integration_tests/abnormal_state.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
@maxi297 I think you mean to delete the `abnormal_state.json` file for `source-zendesk-support` instead of sunshine haha

Reply:
Oops! I'm clearly not the sharpest shooter in the west